src/docs/api/design.mdx

```diff
@@ -0,0 +1,27 @@
+---
+title: "Designing a New API"
+sidebar_order: 2
+---
+Here are some considerations to make when designing new Sentry APIs. 
+
+## URL Patterns
+Your API URL is what developers use to call the endpoint so it’s important to be clear.
+### Don't Exceed 3-level Nesting
+Nested resources format like `api/0/organizations/{org}/projects/` is recommended over `api/0/projects/` for readability because it gives user an understanding of resource hierarchy but they can also make the URLs too long and hard to use. So at Sentry we’re going with 3-level nesting as a hybrid solution. 
+
+Here are some possible urls for values with this resource hierarchy: organization -> project -> tag -> value
+```
```

Comment:
@shanamatthews is there a better way to highlight a block? I used code block but this is not exactly code :/